docs/about/whats-new.ipynb

```diff
@@ -497,6 +600,16 @@
     "    \n",
     "</div>\n",
     "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.8**\n",
+    "\n",
+    "- `to_unit` now has a `copy` argument.\n",
+    "   By default, `copy=True` and `to_unit` makes a copy even if the input already has the desired unit.\n",
+    "   For a cheap way to ensure inputs have expected units use `copy=False` to avoid copies if possible.\n",
+    "    \n",
+    "</div>\n",
+    "\n",
```

Comment:
Similar to above comment. But here the change in behaviour is explained in a separate place from the change in 0.7.